src/sql/workbench/services/connection/browser/connectionWidget.ts

```diff
@@ -127,17 +127,24 @@ export class ConnectionWidget extends lifecycle.Disposable {
 			this._optionsMaps[option.specialValueType] = option;
 		}
 
+		// let's try editing this.
 		let authTypeOption = this._optionsMaps[ConnectionOptionSpecialType.authType];
 		if (authTypeOption) {
-			let authTypeDefault = this.getAuthTypeDefault(authTypeOption, OS);
+			let authTypeDefault = this.getAuthTypeDefault(authTypeOption, OS, this._configurationService);
 			let authTypeDefaultDisplay = this.getAuthTypeDisplayName(authTypeDefault);
 			this._authTypeSelectBox = new SelectBox(authTypeOption.categoryValues.map(c => c.displayName), authTypeDefaultDisplay, this._contextViewService, undefined, { ariaLabel: authTypeOption.displayName });
 		}
 		this._providerName = providerName;
 	}
 
-	protected getAuthTypeDefault(option: azdata.ConnectionOption, os: OperatingSystem): string {
-		// Check for OS-specific default value
+	protected getAuthTypeDefault(option: azdata.ConnectionOption, os: OperatingSystem, configurationService: IConfigurationService): string {
```

Comment:
doesn't the `this` already have an instance of `configurationService`?  why the need for the param also?

Reply:
I'll check and confirm.